### PR TITLE
Separate Settings Form into parts

### PR DESCRIPTION
**Pull request by Jighdan**

### CODE REVIEW

File: src/components/SettingsForm/index.tsx [hunk 1]
```diff
@@ -27,41 +27,33 @@ const SettingsForm = () => {
...
-					<input
-						type="number"
-						min="3"
-						max="50"
-						id="longBreakDuration"
-						value={ secondsToMinutes(durationBreakLong) }
-						onChange={ ({ target }) => setDurationBreakLong(minutesToSeconds(target.valueAsNumber)) }
-					/>
-				</Styled.FormControl>
+				<SettingsFormPomodoroDuration
```
Comment (mrdavid0614) [inline src/components/SettingsForm/index.tsx:40]: I think the name of the component isn't so descriptive

**Review by mrdavid0614 — CHANGES_REQUESTED** [review]

File: src/components/SettingsForm/index.tsx (hunk 1 above)
Comment (Jighdan) [reply src/components/SettingsForm/index.tsx:40]: What do you suggest?

Comment (mrdavid0614) [reply src/components/SettingsForm/index.tsx:40]: I mean, if the component function is an input field, so the name isn't refer to it

Comment (Jighdan) [reply src/components/SettingsForm/index.tsx:40]: I named it thinking of its parent component, being the `SettingsForm`, so I guess the name refers to this component being a child of the said component.

**Pull request merged**